Assets/Global/Scripts/Enemies/FollowEnemy.cs

```diff
@@ -1,38 +1,42 @@
 using System.Collections.Generic;
-using FollowingEnemy;
+using FollowEnemyStates;
 using UnityEngine;
 using UnityEngine.AI;
 
-public class FollowingEnemyScript : EnemyScript
+public class FollowEnemy : EnemyBase
 {
-    public Dictionary<string, FollowingEnemy.BaseState> states;
+    public Dictionary<string, FollowEnemyStates.StateBase> states;
+    private FollowEnemyStates.StateBase currentState;
+    
+    [HideInInspector] public NavMeshAgent agent;
 
-    public FollowingEnemy.BaseState currentState;
-    public NavMeshAgent agent;
-
-    // Roaming behavior
+    [Header("Roaming behavior")]
     public float roamingRadius = 10f;
     public float roamingAngleRange = 90f;
-    public Vector3 roamingNextDestination;
+    // public Vector3 roamingNextDestination; // never used?
```

Comment:
not sure if these should be kept

Reply:
indeed it seems like its no longer needed. I wonder when I did that without realizing.